terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,69 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                  = "" # this must match the stack name the service deploys into
```

Comment:
Stack name required. If the cluster isn't specified here https://companieshouse.atlassian.net/wiki/spaces/Arch/pages/4217208873/ECS+Cluster+Architecture#Search-Service-Cluster then use the same cluster as the web service if it has one please.